tests/expectations/metrics/test_core.py

```diff
@@ -612,6 +615,611 @@ def test_column_histogram_metric_spark(spark_session):
     assert results == {desired_metric.id: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1]}
 
 
+def test_column_partition_metric_pd():
+    week_idx: int
+    engine = build_pandas_engine(
+        pd.DataFrame(
+            {
+                "a": [
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6,
+                    7,
+                    8,
+                    9,
+                    10,
+                    11,
+                ],
+                "b": [
+                    datetime.datetime(2021, 1, 1, 0, 0, 0)
+                    + datetime.timedelta(days=(week_idx * 7))
+                    for week_idx in range(12)
```

Comment:
12 weeks.

Reply:
We will have to find a way to do this in our `DataAssistant` setting in which we read from `csv`.